src/odemis/driver/ueye.py

```diff
@@ -760,10 +766,15 @@ def _set_static_settings(self):
         # Note: for now the default pixel clock seems fine. Moreover, reducing
         # it reduces the minimum exposure time (but we don't care), and the
         # maximum exposure time is constant.
+        self.BlackLevel()
 
-        # Auto black level = different black level per pixel (= more range?)
-        blmode = c_uint32(AUTO_BLACKLEVEL_ON)
-        self._dll.is_Blacklevel(self._hcam, BLACKLEVEL_CMD_SET_MODE, byref(blmode), sizeof(blmode))
+    def BlackLevel(self):
```

Comment:
To me, this looks like this function should be called `SetBlackLevel(blmode)` (and take the mode as argument). Also, move it next to `SetExposureTime()` et al.